app/components/StatusBadge.tsx

```diff
@@ -69,3 +75,17 @@ export const SnapshotStatusBadge = (props: {
     {props.status}
   </Badge>
 )
+
+export const PolicyKindBadge = ({ policy }: { policy: 'in_service' | 'expunged' }) => {
+  const color = policy === 'in_service' ? 'default' : 'neutral'
+  return <Badge color={color}>{policy.replace(/_/g, ' ')}</Badge>
+}
+
+const STATE_BADGE_COLORS: Record<PhysicalDiskState | SledState, BadgeColor> = {
+  active: 'default',
+  decommissioned: 'neutral',
+}
+
+export const StateBadge = ({ state }: { state: PhysicalDiskState | SledState }) => (
+  <Badge color={STATE_BADGE_COLORS[state]}>{state}</Badge>
+)
```

Comment:
This one could be a little simpler if we skipped the STATE_BADGE_COLOR piece of it and just did what we do with PolicyKindBadge where we accept explicit strings.